### PR TITLE
tests fix

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs [hunk 1]
```diff
@@ -110,20 +110,20 @@ public async Task TestGetScoreApi()
...
         public async Task TestGetTransactionResult()
         {
-            var txHash = new Bytes("0xe7ca6280d7de91f33ab3d4fe8359a15fa397c31f8c36a14d48bd995788000374");
+            var txHash = new Bytes("0x0a0972b775a433b591121b4df4ffd8c70fa4c197e28e9e8fb21601f876c9c075");
             var tx = await _iconService.GetTransactionResult(txHash);
             Assert.Equal(txHash, tx.GetTxHash());
         }
 
-        [Fact]
+        [Fact(Skip = "Find transaction with internal transfers")]
```
Comment (KonstantinRyazantsev) [inline tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs]: Why is it skipped?

File: tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs [hunk 2]
```diff
@@ -53,19 +53,19 @@ public async Task TestGetBalance()
             Assert.True(balance >= 0);
         }
 
-        [Fact]
+        [Fact()]
         public async Task TestGetTotalSupply()
         {
             var totalSupply = await _iconService.GetTotalSupply();
-            Assert.Equal(BigInteger.Parse("800460000000000000000000000"), totalSupply);
+            Assert.True(totalSupply > 0);
```
Comment (KonstantinRyazantsev) [inline tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs]: Why not exact value?

File: tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs [hunk 3]
```diff
@@ -18,7 +18,7 @@ namespace Lykke.Icon.Sdk.Tests
 {
     [SuppressMessage("ReSharper", "StringLiteralTypo")]
     [SuppressMessage("ReSharper", "CommentTypo")]
-    public class IconServiceVCRTest : TestWithEnvVariables
+    public class IconServiceVCRTest //: TestWithEnvVariables
```
Comment (KonstantinRyazantsev) [inline tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs]: Commented code here and below

**Review by KonstantinRyazantsev — COMMENTED** [review]
.

File: tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs (hunk 1 above)
Comment (OttoVT) [reply tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs]: I can;t find transaction with internal transfer to check that test

File: tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs (hunk 2 above)
Comment (OttoVT) [reply tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs]: It is increasing per block basis

File: tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs (hunk 3 above)
Comment (OttoVT) [reply tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs]: One day we will revert this(maybe for local testing)

Comment (KonstantinRyazantsev) [reply tests/Lykke.Icon.Sdk.Tests/IconServiceVCRTest.cs]: Not sure about it, but ok :)

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**